packages/dds/tree/src/simple-tree/treeNodeApi.ts

```diff
@@ -122,6 +130,137 @@ export interface TreeNodeApi {
+		data: VerboseTree | undefined,
+		options?: {
+			handleConverter?: undefined;
+			useStableFieldKeys?: boolean;
+		},
+	): TreeFieldFromImplicitField<TSchema>;
+
+	/**
+	 * Like {@link Tree.create}, except deeply clones existing nodes.
+	 * @remarks
+	 * This only clones the persisted data associated with a node.
+	 * Local state, such as properties added to customized schema classes, will not be cloned:
+	 * they will be initialized however they end up after running the constructor, just like if a remote client had inserted the same nodes.
+	 */
+	clone<TSchema extends ImplicitFieldSchema>(
+		original: TreeFieldFromImplicitField<TSchema>,
+		options?: {
+			/**
+			 * If set, all identifier's in the cloned tree (See {@link SchemaFactory.identifier}) will be replaced with new ones allocated using the default identifier allocation schema.
+			 * Otherwise any identifiers will be preserved as is.
+			 */
+			replaceIdentifiers?: true;
+		},
+	): TreeFieldFromImplicitField<TSchema>;
+
+	/**
+	 * Copy a snapshot of the current version of a TreeNode into a JSON compatible plain old JavaScript Object.
+	 *
+	 * @remarks
+	 * If the schema is compatible with {@link ITreeConfigurationOptions.ensureUnambiguous}, then the returned object will be lossless and compatible with {@link Tree.create} (unless the options are used to customize it).
+	 */
+	cloneToJSON<T>(
+		node: TreeNode,
+		options?: { handleConverter(handle: IFluidHandle): T; useStableFieldKeys?: boolean },
+	): JsonCompatible<T>;
+
+	/**
+	 *
+	 */
+	cloneToJSON(
+		node: TreeNode,
+		options?: { handleConverter?: undefined; useStableFieldKeys?: boolean },
+	): JsonCompatible<IFluidHandle>;
+
+	/**
+	 * Copy a snapshot of the current version of a TreeNode into a JSON compatible plain old JavaScript Object.
+	 * Verbose tree format, with explicit type on every node.
+	 *
+	 * @remarks
+	 * If the schema is compatible with {@link ITreeConfigurationOptions.ensureUnambiguous}, then the returned object will be lossless and compatible with {@link Tree.create} (unless the options are used to customize it).
+	 */
+	cloneToJSONVerbose<T>(
+		node: TreeNode,
+		options?: { handleConverter(handle: IFluidHandle): T; useStableFieldKeys?: boolean },
+	): VerboseTree<T>;
+
+	/**
+	 * Same as generic overload, except leaves handles as is.
+	 */
+	cloneToJSONVerbose(
+		node: TreeNode,
+		options?: { handleConverter?: undefined; useStableFieldKeys?: boolean },
+	): VerboseTree;
+}
+
+export type VerboseTreeValue<THandle = IFluidHandle> =
+	| VerboseTree<THandle>
+	| Exclude<IFluidHandle, TreeLeafValue>
+	| THandle;
+
+/**
+ * The fields required by a node in a tree.
+ *
+ * @privateRemarks A forked version of this type is used in `persistedTreeTextFormat.ts`.
+ * Changes to this type might necessitate changes to `EncodedNodeData` or codecs.
+ * See persistedTreeTextFormat's module documentation for more details.
+ *
+ * @public
+ */
+export interface VerboseTree<THandle = IFluidHandle> {
```

Comment:
This is a proposed simple tree explicitly types format. Its less verbose than jsonable tree due to not boxing leaf values and making fields an array or a record not a record of arrays.

@Josmithr : I think you may have already been defining a type like this for your use?

Reply:
I didn't define a type like this directly, but have defined a JSON Schema subset that depends implicitly on a representation like this. 
Draft PR: https://github.com/microsoft/FluidFramework/pull/21984/files#diff-5f88b1ac2dd53efb279ea3f0e6e8124af3e6d3fefdac113ee129959855dfb5b5